tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -621,6 +621,9 @@ declare namespace ts {
         type?: TypeNode;
         initializer?: Expression;
     }
+    interface PrivatePropertyDeclaration extends PropertyDeclaration {
```

Comment:
since this is public API:
- maybe we don't need another interface for this
- if we keep it, `PrivateNamedPropertyDeclaration` is probably more accurate, since there are already private properties